app-text/poppler/metadata.xml

```diff
@@ -11,4 +11,8 @@
 	<flag name='introspection'>Enable GObject introspection.</flag>
 	<flag name='utils'>Install command-line PDF converters and various utilities.</flag>
 </use>
+<slots>
+	<subslots>Reflects ABI compatibility of libpoppler.so only,
```

Comment:
I'd prefer to stick with `ABI` wording, since subslot changes probably include SOVERSION changes, but can potentially also go beyond that in case upstream messed up.